docs/source/intervention_models/crm_mgmt/index.rst

```diff
@@ -80,21 +79,35 @@ treatment, which is known to increase adherence.
     - Effect
     - Modeled?
     - Note (ex: is this relationship direct or mediated?)
-  * - 
-    - 
-    - 
-    - 
-  * - 
-    - 
-    - 
-    - 
+  * - SBP
+    - additive shift
+    - yes
+    - there is a “ramp” of increasing intensity of treatments
+  * - LDL-C
+    - multiplicative shift
+    - yes
+    - only two values: on treatment or not on treatment
+  * - BMI
+    - additive?
+    - yes
+    - [[need more detail on this effect]]
+  * - FPG
+    - additive?
+    - yes
+    - [[need more detail on this effect]]
+  * - Tobacco
```

Comment:
Is `Tobacco use` the only discontinuous exposure variable?

Reply:
TBH, I don't know what sort of Tobacco Use risk exposure is planned for this model. It is [mentioned 5 times in the concept model document and shown clearly in the concept model diagram](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_us_cvd/concept_model.html), but I don't know if this model needs [any of the smoking status, pack years, or years since quitting exposures we have done before](https://vivarium-research.readthedocs.io/en/latest/gbd2017_models/risk_exposure/smoking/index.html) or something new and different.

Reply:
The level 3 risk of Tobacco use includes 1) chewing tobacco, and 2) smoking. We don't have a risk exposure and risk effect page for level 2 tobacco use,  methods from GBD can be found here:
https://hub.ihme.washington.edu/display/TEAM/Modeling+Tobacco+Step+by+Step-+START+HERE